test/__init__.py

```diff
@@ -139,6 +140,17 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def requires_bundled_OpenSSL_on_mac(test):
+    """Skips this test unless your MacPython bundles OpenSSL"""
+
+    @functools.wraps(test)
+    def wrapper(*args, **kwargs):
+        if platform.system() == 'Darwin' and sys.version_info[0:2] < (3, 6):
```

Comment:
Should we allow Python 2.7 as well per your comment?

Reply:
Thanks! Fixed.